mage_ai/frontend/components/CodeBlock/index.tsx

```diff
@@ -240,6 +240,16 @@ function CodeBlock({
   const [content, setContent] = useState(defaultValue);
   const [currentTime, setCurrentTime] = useState<number>(null);
 
+  const isStreamingPipeline = PipelineTypeEnum.STREAMING === pipeline?.type;
+  const isDBT = BlockTypeEnum.DBT === block?.type;
+  const isSQLBlock = BlockLanguageEnum.SQL === block?.language;
+  const isRBlock = BlockLanguageEnum.R === block?.language;
+
+  let defaultLimitValue = blockConfiguration[CONFIG_KEY_LIMIT];
+  if (!isDBT && isSQLBlock && defaultLimitValue == null) {
```

Comment:
Shouldn’t we check for undefined?

Reply:
it checks for both null and undefined, but changed it to just undefined